### PR TITLE
Add Tabbed Application Mode article

### LINKED ISSUE #4484: content: Tabbed Application Mode [2021-02-28]
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/qS3GB-4c7Vk)
* draft: TBD

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #4484

Changes proposed in this pull request:

- Add the article.

### CODE REVIEW

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+{% Aside %}
+  Tabbed application mode is part of the
+  [capabilities project](https://web.dev/fugu-status/) and is currently in development. This post
+  will be updated as the implementation progresses. Tabbed application mode is an early-stage
+  exploration of the Chrome team. It is not ready for production yet.
+{% endAside %}
+
+In the world of computing, the [desktop metaphor](https://en.wikipedia.org/wiki/Desktop_metaphor) is
+an interface metaphor which is a set of unifying concepts used by graphical user interfaces (GUI) to
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/tabbed-application-mode/index.md:26]: In Am English, 'that' is preferred for a restrictive clause, which this is. 

Or

'That' is preferred for the restrictive clause that this is.

The first sentence is about all restrictive clauses, so 'which' and a comma are used to indicate the subordinate clause does not refer to all restrictive clauses. The second phrasing is only about your sentence, so 'that' is used to indicate that the subordinate clause is restricting the number of things we're talking about.

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+## What is tabbed application mode
+
+Something that has been missing from the platform so far is a way to let PWA developers offer their
+users a tabbed document interface, for example, to enable editing different files in the same PWA
+window. Tabbed application mode closes this gap.
+
+{% Aside %}
+  This feature is about having a standalone app window with multiple tabs (containing
+  separate documents inside the app scope) inside it. It is not to be confused with
+  `"display": "browser"`, which does already exist and has a separate meaning (specifically, that
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:49]:
```suggestion
  the existing `"display": "browser"`, which has a separate meaning (specifically, that
```

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  the app is opened in a regular browser tab).
+{% endAside %}
+
+### Suggested use cases for tabbed application mode
+
+Examples of sites that may use tabbed application mode include:
+
+- Productivity apps that let the user edit more than one document (or file) at the same time.
+- Communication apps can let the user have conversations in different rooms per tab.
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:58]:
```suggestion
- Communication apps that let the user have conversations in different rooms per tab.
```

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+- Reading apps can open article links in new in-app tabs.
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:59]:
```suggestion
- Reading apps that open article links in new in-app tabs.
```

File: src/site/content/en/blog/tabbed-application-mode/index.md (hunk 3 above)
Comment (jpmedley) [reply src/site/content/en/blog/tabbed-application-mode/index.md:58]: Somewhere there's some writing advice about mixing up wording to keep a piece from being monotonous. In the case of lists, the rule to use parallel grammatical structures has priority. (Structures can be repeated for rhetorical effect too. See [the second paragraph of this](https://www.archives.gov/founding-docs/declaration-transcript).

Comment (jpmedley) [reply src/site/content/en/blog/tabbed-application-mode/index.md:58]: That link predates most style guides and modern developments in language usage, so it seems to 'break' rules that I've been trying to share.

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## Using tabbed application mode
+
+To use tabbed application mode, developers need to opt in their apps by setting a specific display
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:102]:
```suggestion
To use tabbed application mode, developers need to opt their apps in by setting a specific display
```

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+### Trying tabbed application mode
+
+You can try tabbed application mode on Chrome&nbsp;OS devices running Chrome&nbsp;83 and up today:
+
+1. Set the `#enable-desktop-pwas-tab-strip` flag.
+1. Install any web app that runs in `standalone` mode, for example,
+   [Excalidraw](https://excalidraw.com/).
+1. Pin the app icon to the shelf, right click the icon, and select "New tabbed window" from the
+   contextmenu.
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:125]:
```suggestion
   context menu.
```

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+### Report a problem with the implementation
+
+Did you find a bug with Chrome's implementation? File a bug at
+[new.crbug.com](https://new.crbug.com). Be sure to include as much detail as you can, simple
+instructions for reproducing, and enter `UI>Browser>WebAppInstalls` in the **Components** box.
+[Glitch](https://glitch.com/) works great for sharing quick and easy reproduction cases.
+
+### Show support for the API
+
+Are you planning to use tabbed application mode? Your public support helps the Chrome team to
```
Comment (jpmedley) [inline src/site/content/en/blog/tabbed-application-mode/index.md:151]:
```suggestion
Are you planning to use tabbed application mode? Your public support helps the Chrome team
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/tabbed-application-mode/index.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: Tabbed application mode for PWAs
+subhead: Work on more than one document at a time with tabs in your Progressive Web App
+authors:
+  - thomassteiner
+date: 2021-02-16
```
Comment (tomayac) [inline src/site/content/en/blog/tabbed-application-mode/index.md:7]:
```suggestion
date: 2021-02-25
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**